eo-runtime/src/test/java/org/eolang/PhDefaultTest.java

```diff
@@ -29,6 +29,11 @@ final class PhDefaultTest {
      */
     private static final String VOID_ATT = "void";
 
+    /**
+     * Name of context.
+     */
+    private static final String CONTEXT = "context";
```

Comment:
@karishka1222 avoid static literals in the tests, instead - try to make all your test inputs - unique. Also read this: https://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html